gocd/templates/libs/relay-pops.libsonnet

```diff
@@ -11,6 +11,10 @@ local region_pops = {
     'us-pop-2',
     'us-pop-3',
     'us-pop-4',
+    'us-pop-regional-1',
```

Comment:
why are they all us?
should it be asia, australia, europe and then us here?

Reply:
that's how the files are named ~ https://github.com/getsentry/devinfra-deployment-service/pull/513/files